### PR TITLE
busybox: prepare busybox for running test suite

### PR DESCRIPTION
**Pull request by damianloew**
JIRA: RTOS-59

## Description

-Copy testsuite to /usr/test/busybox
-Copy busybox .config file to bin directory (some busybox tests use that file)
-Update busybox config file 

## Motivation and Context

It's needed for running busybox testsuite from test_busybox in phoenix-rtos-tests.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [x] Tested by hand on: (ia32-generic: qemu).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Comment by damianloew** [comment]
https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/34

**Comment by damianloew** [comment]
https://github.com/phoenix-rtos/phoenix-rtos-build/pull/75

**Review by nalajcie — COMMENTED** [review]
Tested on `imx6ull` (without C wrapper program), the results are similar do the ones on `ia32-generic`.

File: busybox/build.sh [hunk 1]
```diff
@@ -52,3 +52,10 @@ fi
 make -C "${PREFIX_BUSYBOX_BUILD}" KBUILD_SRC="$PREFIX_BUSYBOX_SRC" -f "${PREFIX_BUSYBOX_SRC}"/Makefile CROSS_COMPILE="$CROSS" CONFIG_PREFIX="$PREFIX_FS/root" SKIP_STRIP="$UNSTRIPPED" all
 make -C "${PREFIX_BUSYBOX_BUILD}" KBUILD_SRC="$PREFIX_BUSYBOX_SRC" -f "${PREFIX_BUSYBOX_SRC}"/Makefile CROSS_COMPILE="$CROSS" CONFIG_PREFIX="$PREFIX_FS/root" SKIP_STRIP="$UNSTRIPPED" install
 cp -a "$PREFIX_BUSYBOX_BUILD/busybox_unstripped" "$PREFIX_PROG"
+
+if [ "$B_PHOENIX_LONG_TESTS" = "y" ]; then
+mkdir -p "$PREFIX_FS/root/bin/scripts"
```
[the remark this answers is not in the document]
Comment (damianloew) [reply busybox/build.sh]: Thanks, I realize that we don't need it, corrected

**Review by nalajcie — APPROVED** [review]

**Pull request merged**